tidy3d/plugins/dispersion/__init__.py

```diff
@@ -1,4 +1,4 @@
 """ Imports from dispersion fitter plugin. """
 
 from .fit import DispersionFitter
-from .fit_web import StableDispersionFitter, AdvancedFitterParam
+from .web import AdvancedFitterParam, StableDispersionFitter
```

Comment:
does this break backwards compatibility for the Fitter notebook? if so, maybe we could leave `fit_web.py` with just an import of these two classes.

Reply:
Good catch. To be safe, we can import everything from "web.py" and issue a deprecation warning. What do you think?
The notebook itself works fine (with a deprecation warning) because it doesn't import "fit_web.py" directly, but users might.